Orange/widgets/data/owdatasampler.py

```diff
@@ -294,62 +293,100 @@ def send_report(self):
         self.report_items(items)
 
 
-def sample_fold_indices(table, folds=10, stratified=False, random_state=None):
-    """
-    :param Orange.data.Table table:
-    :param int folds: Number of folds
-    :param bool stratified: Return stratified indices (if applicable).
-    :param Random random_state:
-    :rval tuple-of-arrays: A tuple of array indices one for each fold.
-    """
-    if stratified and table.domain.has_discrete_class:
-        # XXX: StratifiedKFold does not support random_state
-        ind = skl_cross_validation.StratifiedKFold(
-            table.Y.ravel(), folds, random_state=random_state)
-    else:
-        ind = skl_cross_validation.KFold(
-            len(table), folds, shuffle=True, random_state=random_state)
-    return tuple(ind)
-
-
-def sample_random_n(table, n, stratified=False, replace=False,
+class sample_fold_indices():
+    def __init__(self, folds=10, stratified=False, random_state=None):
+        """
+        :param int folds: Number of folds
+        :param bool stratified: Return stratified indices (if applicable).
+        :param Random random_state:
+        :rval tuple-of-arrays: A tuple of array indices one for each fold.
```

Comment:
Since this is new code, could you also make this a [NumPy or Google style docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#google-vs-numpy)?